.github/workflows/azure-dev.yml

```diff
@@ -1,6 +1,14 @@
 name: Deploy
 
 on:
+  workflow_call:
+    inputs:
+      deploy-only:
+        description: 'only run azd deploy with no provisioning'
+        required: true
+        default: false
+        type: boolean
```

Comment:
Once again, due to the way ARM works, this doesn't make much sense: https://github.com/ucl-contoso-chat/ucl-openai-search/pull/11#issuecomment-2230716330

Reply:
same above